jwala-tomcat/build.gradle

```diff
@@ -1,176 +1,177 @@
-description "This module generates and deploys Jwala binaries. This layes creates the zip file which can be distributed and published to a repository for releases"
```

Comment:
Why does git flag the whole content ? Can you double check hidden characters ? e.g. tabs = 4 spaces